azure-resource-optimization-toolkit/azuredeploy.json

```diff
@@ -0,0 +1,112 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "newGuid": {
+      "type": "string",
+      "metadata": {
+        "description": "Enter new GUID, you can generate one from Powershell using new-guid or get one from this site: <a target=new href=https://guidgenerator.com/online-guid-generator.aspx>GUID Generator</a>"
+      }
+    },
+    "AutomationAccountName": {
+      "type": "string",
+      "metadata": {
+        "description": "Assign a name for the Automation account of your choosing.  Must be a unique name as Azure Automation accounts are FQDNs"
+      }
+    },
+    "AutomationRegion": {
+      "type": "string",
+      "defaultValue": "East US 2",
+      "allowedValues": [
+        "Japan East",
+        "East US 2",
+        "West Europe",
+        "Southeast Asia",
+        "South Central US",
+        "North Europe",
+        "Canada Central",
+        "Australia Southeast",
+        "Central India",
+        "Japan East"
+      ],
+      "metadata": {
+        "description": "Specify the region for your Automation account"
+      }
+    },
+    "azureAdmin": {
+      "type": "string",
+      "metadata": {
+        "description": "Enter your service admin user, ex: serviceaccount@microsoft.com.  Must be administrator on the subscription you're deploying too during template deployment."
+      }
+    },
+    "azureAdminPwd": {
+      "type": "securestring",
+      "metadata": {
+        "description": "Enter the password for the service admin user. The pwd is encrypted during runtime and in the Automation assets"
+      }
+    },
+    "_artifactsLocation": {
```

Comment:
I know this is a common pattern for templates in this repo - but it's an antipattern. Can you switch to putting this into a variable defined like this: "[uri(deployment().properties.templateLink.uri, '/nestedtemplates/template.json')]"